src/Fmt_ast.ml

```diff
@@ -726,19 +729,20 @@ and fmt_core_type c ?(box= true) ?(in_type_declaration= false) ?pro
       hvbox 0
         (wrap "< " " >"
            ( list fields "@ ; " (function
-               | Otag (lab_loc, attrs, typ) ->
-                   (* label loc * attributes * core_type -> object_field *)
-                   let doc, atrs = doc_atrs attrs in
-                   let fmt_cmts = Cmts.fmt c.cmts lab_loc.loc in
-                   fmt_cmts
-                   @@ hvbox 4
-                        ( hvbox 2
-                            ( Cmts.fmt c.cmts lab_loc.loc @@ str lab_loc.txt
-                            $ fmt ":@ "
-                            $ fmt_core_type c (sub_typ ~ctx typ) )
-                        $ fmt_docstring c ~pro:(fmt "@;<2 0>") doc
-                        $ fmt_attributes c ~pre:(fmt " ") ~key:"@" atrs )
-               | Oinherit typ -> fmt_core_type c (sub_typ ~ctx typ) )
```

Comment:
Here too, the red indentation is preferred?

Reply:
yes